README.md

```diff
@@ -1 +1,17 @@
-# react-native-with-moxy
\ No newline at end of file
+# react-native-with-moxy
+MOXY's boilerplate to accelerate the setup of new [React Native](https://reactnative.dev/) based applications.
```

Comment:
Please keep new-lines consistent everywhere. Please check in other files as well.